src/input/plugins/CurlInputPlugin.cxx

```diff
@@ -309,7 +311,7 @@ CurlInputStream::OnData(ConstBuffer<void> data)
 
 	if (data.size > GetBufferSpace()) {
 		AsyncInputStream::Pause();
-		throw CurlResponseHandler::Pause{};
+		throw CurlRequest::Pause();
```

Comment:
Why this change?

Reply:
Sorry, started with 0.22.4, switched to main, so there is this change in the last commit